ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -0,0 +1,384 @@
+                    asset.name: asset.emoji || "filled-account"
+                    asset.isLetterIdenticon: asset.emoji
+                    asset.letterSize: 14
+                    asset.bgColor: Theme.palette.primaryColor3
+                    asset.isImage: asset.emoji
+                    components: [
+                        StatusIcon {
+                            anchors.verticalCenter: parent.verticalCenter
+                            visible: model.walletType === Constants.watchWalletType
+                            icon: "show"
+                            color: Theme.palette.directColor1
+                        },
+                        StatusFlatButton {
+                            anchors.verticalCenter: parent.verticalCenter
+                            size: StatusBaseButton.Size.Small
+                            enabled: !accountDelegate.saved
+                            text: accountDelegate.saved ? qsTr("Address saved") : qsTr("Save Address")
+                            onClicked: {
+                                accountDelegate.saved = root.walletStore.createOrUpdateSavedAddress(model.name, model.address, false) === ""
+                                Global.displayToastMessage(qsTr("%1 saved to your wallet").arg(accountDelegate.subTitle),
+                                                           qsTr("Go to your wallet"),
+                                                           "wallet",
+                                                           false,
+                                                           Constants.ephemeralNotificationType.normal,
+                                                           `#${Constants.appSection.wallet}` // internal link to wallet section
+                                                           )
+                            }
+                        },
+                        StatusFlatRoundButton {
+                            anchors.verticalCenter: parent.verticalCenter
+                            type: StatusFlatRoundButton.Type.Secondary
+                            icon.name: "send"
+                            tooltip.text: qsTr("Send")
+                            onClicked: {
+                                root.walletStore.switchAccountByAddress(model.address)
+                                Global.openSendModal(model.address)
+                            }
+                        },
+                        StatusFlatRoundButton {
+                            anchors.verticalCenter: parent.verticalCenter
+                            type: StatusFlatRoundButton.Type.Secondary
+                            icon.name: "copy"
+                            tooltip.text: qsTr("Copy")
+                            onClicked: {
+                                tooltip.text = qsTr("Copied")
+                                root.profileStore.copyToClipboard(model.address)
+                                d.timer.setTimeout(function() {
```

Comment:
Maybe we could exclude some utility component for stuff like that:

```
ExpiringBinding on tooltip.text {
  id: copiedBinding
  value: qsTr("Copied")
  timeout: 2000
}

and in onClicked:
```
  copiedBinding.start()
```
than there is no need to repeat literal `Copy`.
}

Reply:
I agree